.github/workflows/build.yml

```diff
@@ -108,10 +108,10 @@ jobs:
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
       - name: Setup Minikube
-        uses: manusa/actions-setup-minikube@v2.4.2
+        uses: manusa/actions-setup-minikube@v2.7.1
         with:
-          minikube version: 'v1.24.0'
-          kubernetes version: 'v1.22.4'
+          minikube version: 'v1.26.1'
+          kubernetes version: 'v1.25.0'
```

Comment:
https://github.com/kubernetes/kubernetes/releases/tag/v1.25.4

Reply:
Can we use more recent versions that are mentioned in https://github.com/manusa/actions-setup-minikube?